packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -265,10 +361,6 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
     const roundedValue = parseFloat(value.toFixed(numDec));
     const valueChanged = roundedValue !== this.state.value;
 
-    if (snapToStep) {
```

Comment:
Are we not using the `snapToStep` prop anymore?

Reply:
oh I ve readded snaps